.github/workflows/pre-commit.yml

```diff
@@ -0,0 +1,37 @@
+name: pre-commit
+
+on:
+  pull_request:
+    branches:
+      - "13.0*"
+  push:
+    branches:
+      - "13.0"
+      - "13.0-ocabot-*"
```

Comment:
I think it's master here.

Reply:
Yes.

Reply:
This was the key to execute the GH workflow!